modular-docs-manual/files/sample-assemblies/.gitignore

```diff
@@ -0,0 +1 @@
+**/build/
```

Comment:
@emmurphy1 , overall comment: We've discussed this already so apologies if it's a dead horse. :) I wouldn't name the overall folder `sample-assemblies` because it's really a sample repo, much like the `simple-mod-guide-repo.zip` that already exists here, but for a different use case. The existing repo example demonstrates a repo where assemblies are generated within a single running doc (single master.adoc). Your repo example demonstrates a repo where assemblies are generated in multiple separate docs, each assembly its own doc (multiple master.adocs). So I would call out that fact in the example, in a description somewhere, and would rename the overall folder to `intermediate-mod-guide-repo` for consistency with the other example. Also might be good to likewise ZIP the folder and deliver as a zip file like the other? If not, we should have someone unzip the other example so the two are parallel. Just a thought.